egasub/submission/submittable/alignment.py

```diff
@@ -34,3 +40,7 @@ def local_validate(self,ega_enums):
         if not any(cc['tag'] == str(self.sample.case_or_control_id) for cc in ega_enums.lookup("case_control")):
             self._add_local_validation_error("sample",self.sample.alias,"caseOrControl","Invalid value '%s'" % self.sample.case_or_control_id)
 
+    def ftp_files_remote_validate(self,host,username, password):
+        for _file in self._analysis.files:
+            if not file_exists(host,username,password,_file.file_name):
+                self._add_ftp_file_validation_error("analysis",self.analysis.alias,"fileName","File missing on FTP ega server: %s" % _file.file_name)
```

Comment:
only need two parameters here: `file_name`, and `error` (the text of error message)